internal/data/data.go

```diff
@@ -75,7 +77,8 @@ func NewCkbNodeClient(conf *config.CkbNode, logger *logger.Logger) (*CkbNodeClie
 		return nil, err
 	}
 	return &CkbNodeClient{
-		Rpc: client,
+		Rpc:  client,
+		Mode: conf.Mode,
```

Comment:
The usage of `Mode` is not seen below

Reply:
There is no need to use it yet, but this property is needed.